melosys-eessi-app/src/main/java/no/nav/melosys/eessi/service/joark/JournalpostService.java

```diff
@@ -25,6 +25,13 @@ OpprettJournalpostResponse opprettInngaaendeJournalpost(SedHendelse sedHendelse,
         return opprettJournalpost(request, false);
     }
 
+    OpprettJournalpostResponse opprettOgFerdigstillInngaaendeJournalpost(SedHendelse sedHendelse, Sak sak,
+            SedMedVedlegg sedMedVedlegg) {
+        OpprettJournalpostRequest request = OpprettJournalpostRequestMapper.opprettInngaaendeJournalpost(
+                sedHendelse, sedMedVedlegg, sak, dokkatService.hentMetadataFraDokkat(sedHendelse.getSedType()), null);
```

Comment:
Hvordan kan man ferdigstille journalføring uten ident? https://confluence.adeo.no/display/BOA/opprettJournalpost

Reply:
Kanskje sette bruker som ukjent eller lignende om den mangler?

Reply:
> Kanskje sette bruker som ukjent eller lignende om den mangler?

Hvis det går an må det kanskje avtales med #team_dokumentløsninger?

Reply:
Kom frem til å ikke forskjellsbehandle X100 SEDer. Litt usikkerhet rundt hva som var grunnen til at det ble gjort i det første og melosys burde egentlig ha støtte for dem.
Reverter committen og fjerner filtrering på X100. Hvis ting feiler fikser jeg i en annen branch